services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -83,6 +80,28 @@ export const ChoiceListField = ({
     });
   };
 
+  const clearNestedValues = (choices: FieldChoice[]) => {
+    return choices.forEach((choice: FieldChoice) => {
+      // if a choice is not selected and there are children, clear out any saved data
+      if (!choice.checked && choice.children) {
+        choice.children.map((child) => {
```

Comment:
Nit: `.forEach` is maybe a better choice here because we don't need the output array that it is creating. `.forEach` is a bit cleaner in that way and this may be contributing to CC's "cognitive complexity score" or whatever.